collect_app/src/main/java/org/odk/collect/android/widgets/AbstractSelectOneWidget.java

```diff
@@ -53,20 +47,22 @@
  * @author Yaw Anokwa (yanokwa@gmail.com)
  */
 @SuppressLint("ViewConstructor")
-public abstract class AbstractSelectOneWidget extends SelectTextWidget
-        implements OnCheckedChangeListener, AudioPlayListener, MultiChoiceWidget, View.OnClickListener {
+public abstract class AbstractSelectOneWidget extends SelectTextWidget implements MultiChoiceWidget {
+
+    /**
+     * An estimated max number of elements for whom we don't need to resize a RecyclerView
```

Comment:
I think there needs to be a bit more context here. Maybe something like:
"A list of choices can have thousands of items. To increase loading and scrolling performance, a RecyclerView is used. Because it is nested inside a ScrollView, by default, all of the RecyclerView's items are loaded and there is no performance benefit over a ListView. This constant is used to bound the number of items loaded. The value 40 was chosen because it is around the maximum number of elements that can be shown on a large tablet."

Reply:
Done.